language/stdlib/modules/LibraConfig.move

```diff
@@ -224,6 +224,12 @@ module LibraConfig {
 
         let config_ref = borrow_global_mut<Configuration>(CoreAddresses::LIBRA_ROOT_ADDRESS());
         let current_time = LibraTimestamp::now_microseconds();
+
+        // Do not do anything if a reconfiguration event is already emitted within this transaction.
+        if (current_time == config_ref.last_reconfiguration_time) {
+            return ()
+        };
+
```

Comment:
Is the early return at the start of this function needed still after this change?

Reply:
This will not be needed. Although I'm trying to minimize the change at the moment. To get rid of those lines we will need to reorganize the genesis code a little bit, which I'm not planning to work on in this PR.